Packs/CommonTypes/ReleaseNotes/3_9_0.md

```diff
@@ -0,0 +1,41 @@
+
+#### Incident Fields
+
+##### Destination IPs
+
+Added the `CrowdStrike Falcon NGSIEM Detection` incident type as an associated type.
+
+##### Display Name
+
+- Added the `CrowdStrike Falcon NGSIEM Detection` incident type as an associated type.
+- Added the `CrowdStrike Falcon Third Party Detection` incident type as an associated type.
+
+##### Username
+
+Added the `CrowdStrike Falcon Third Party Detection` incident type as an associated type.
+
+##### Source IPV6
+
+Added the `CrowdStrike Falcon Third Party Detection` incident type as an associated type.
+
+##### Last Update Time
+
+- Added the `CrowdStrike Falcon NGSIEM Detection` incident type as an associated type.
+- Added the `CrowdStrike Falcon Third Party Detection` incident type as an associated type.
+
+##### End Time
+
+Added the `CrowdStrike Falcon NGSIEM Detection` incident type as an associated type.
```

Comment:
Added the **CrowdStrike Falcon NGSIEM Detection** incident type as an associated type.